apps/menu-bar/.eslintrc.js

```diff
@@ -1,7 +1,6 @@
 module.exports = {
   root: true,
-  extends: '@react-native-community',
-  plugins: ['react-hooks'],
```

Comment:
can we keep the `react-hooks` plugins? This is quite useful for things like missing dependencies in useEffects
![image](https://github.com/expo/orbit/assets/11707729/94dbfe03-61a7-40ab-b288-f91d6c0758ff)

Reply:
Hooks plugin is a part of Universe config:
* https://github.com/expo/expo/blob/a95e899e0408c4382d9c198cae9bb012e53bda98/packages/eslint-config-universe/package.json#L50-L50
* https://github.com/expo/expo/blob/a95e899e0408c4382d9c198cae9bb012e53bda98/packages/eslint-config-universe/shared/react.js#L3

Reply:
Is that working for you? Warnings are not showing up on my machine

Reply:
This exact rule is turned off by default in the preset:
* https://github.com/expo/expo/blob/a95e899e0408c4382d9c198cae9bb012e53bda98/packages/eslint-config-universe/shared/react.js#L40

We can enable it manually for menu-bar app.